src/Uno.UI.RemoteControl/RemoteControlClient.cs

```diff
@@ -76,7 +77,7 @@ private async Task StartConnection()
 			try
 			{
 #if __WASM__
-				var isHttps = WebAssemblyRuntime.InvokeJS("window.location.protocol == 'https:'").ToLower() == "true";
+				var isHttps = WebAssemblyRuntime.InvokeJS("window.location.protocol == 'https:'").Equals("true", StringComparison.OrdinalIgnoreCase);
```

Comment:
I could do `.ToLower(CultureInfo.InvariantCulture)`, but this allocates a new string for no reason. String comparison should be better here.

Reply:
`bool.TryParse` seems significantly slower.

![image](https://user-images.githubusercontent.com/31348972/199830183-9d41d7d8-5586-4ac5-8c9b-ce76975a012f.png)

Reply:
Interesting :-) 

I don't recall where (or when) I read that... it might be on 7.0 or it just not as fast anyway since it has to return 2 values (but it might faster than previous net releases). Anyway I doubt it would make a difference when the code is about to invoke some javascript :)

Reply:
That particular code is only called once, in debug, for javascript invocation for which the interop will take signficantly more than either solution ;) In that case, (and martin may remember why he wrote it that way), using tryparse is more readable.